src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1183,15 +1210,11 @@
         message))))
 
 (schema/defn ^:always-validate delete-certificate!
```

Comment:
Would it be worth adding the cases (can't delete, doesn't exist) from the csr deletion to this function in a followup ticket?